src/experimental-layers/src/path-marker-layer/path-marker-layer.js

```diff
@@ -44,12 +44,24 @@ export default class PathMarkerLayer extends CompositeLayer {
     };
   }
 
+  projectFlat(xyz, viewport, coordinateSystem, coordinateOrigin) {
```

Comment:
Perfect, thanks! Could even go into its own file to further mark it for reuse/upstreaming...